MAUI/Scheduler/resource-view.md

```diff
@@ -0,0 +1,375 @@
+---
+layout: post
+title: Resource in MAUI Scheduler control | Syncfusion
+description:  Learn here all about grouping the appointments based on the resources in the timeline day, timeline week, timeline workweek and timeline month views in Syncfusion .NET MAUI Scheduler (SfScheduler) control.
+platform: Maui
+control: SfScheduler
+documentation: ug
+---
+
+# Resource in MAUI Scheduler (SfScheduler)
+
+The .NET MAUI Scheduler control allows you to group appointments based on the resources associated with them in the timeline day, timeline week, timeline workweek, and timeline month views, with complete business object binding, multi resource appointment sharing and UI customization features.
+
+## Create resources to Scheduler by using SchedulerResource
+You can create a resource view by setting the Name, Id, Background, and Foreground properties of the built-in SchedulerResource class and assign SchedulerResource collection to the scheduler by using the Resources property of the ResourceView class.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="3" %}
+<schedule:SfScheduler x:Name="Scheduler"  View="TimelineWeek">
+</schedule:SfScheduler>
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="9" %}
+// Adding schedule resource in the scheduler resource collection.
+var resourceCollection = new ObservableCollection<SchedulerResource>()
```

Comment:
rename property name resourceCollection to resources , do the changes in all places